usaspending_api/config/envs/local.py

```diff
@@ -54,6 +54,11 @@ class LocalConfig(DefaultConfig):
     ES_HOST: str = "localhost"
     ES_PORT: str = "9200"
 
+    # ==== [Spark] ====
+    # Sensible defaults to underneath the project root dir. But look in .env for overriding of these
+    SPARK_SQL_WAREHOUSE_DIR: str = str(_PROJECT_ROOT_DIR / "spark-warehouse")
+    HIVE_METASTORE_DERBY_DB_DIR: str = str(_PROJECT_ROOT_DIR / "spark-warehouse" / "metastore_db")
```

Comment:
Shouldn't these also be added to default and qat config?

Reply:
No, because I think we only want to self-manage the hive metastore_db and spark-warehouse when running in a _local_ environment. 

In those AWS-hosted environments, the Spark platform takes care of that.